log.cpp

```diff
@@ -235,13 +235,13 @@ static void crash_handler(int signo, siginfo_t* si, void*) {
   }
 
   if (si) {
```

Comment:
I suggest `#if defined(__sun) && !defined(HAVE_DPRINTF)` and adding dprintf to ac_check_funcs just in case solaris gets dprintf. It is after all a posix.1-2008 function.